packages/agents/src/functions/ExecuteScript.ts

```diff
@@ -45,16 +45,16 @@ export class ExecuteScriptFunction extends AgentFunctionBase<ExecuteScriptFuncPa
     }
   }
 
-  buildExecutor(agent: Agent<unknown>, context: AgentBaseContext): (params: ExecuteScriptFuncParameters) => Promise<AgentFunctionResult> {
-    return async (params: ExecuteScriptFuncParameters): Promise<AgentFunctionResult> => {
+  buildExecutor(agent: Agent<unknown>, context: AgentBaseContext): (params: ExecuteScriptFuncParameters, rawParams: string | undefined) => Promise<AgentFunctionResult> {
+    return async (params: ExecuteScriptFuncParameters, rawParams: string | undefined): Promise<AgentFunctionResult> => {
```

Comment:
wouldn't this `rawParams` overwrite the one defined at `buildExecutor`?

Reply:
No, it shouldn't unless I made a mistake somewhere. It's just an additional method argument. It's also added to the `buildExecutor` definition.